### PR TITLE
Login Items for Griffins

### PR DESCRIPTION
**Pull request by winishahines**
@davidsbao I've documented the we discussed. Feel free to edit and merge once you've had a chance to review with the other team leads.

### CODE REVIEW

**Comment by davidsbao** [comment]
@chrisj-usds Can you take a quick look at this before I merge these updates in? Just documenting a few login related decisions that we've made over time.

**Comment by chrisj-usds** [comment]
@davidsbao your changes lgtm

**Review by davidsbao — APPROVED** [review]
Approved after review by @chrisj-usds

**Pull request merged**
